.github/template-contributorIntro.md

```diff
@@ -0,0 +1,13 @@
+**Issue Title:** New Contributor Self-Intro [your real name]
+
+Copy-and-paste the following content to the issue tracker and fill in with relevant details:
+```markdown
+- **Link to your app:** e.g. https://teammates-username.appspot.com
+- **Link to your fork:** e.g. https://github.com/username/teammates
+
+**Post a screenshot of changes you did (as seen on the UI).**
+
+
+
+**Tell us your real name and give a short self-introduction, e.g. your education background, why you want to contribute to TEAMMATES, etc.**
```

Comment:
`Tell us your real name and give a short self-introduction (e.g. your education background, why you want to contribute to TEAMMATES, etc)`

Suggest formatting it in this way.